### PR TITLE
Add text area component to style guide

### PR DESCRIPTION
**Pull request by mmlumba**
screen shot 2017-05-10 at 9 50 33 am
Connects #1409

### CODE REVIEW

**Comment by mmlumba** [comment]
@gnakm please confirm if the label in the screenshot is alright or if you'd like me to change it to something else

File: app/assets/stylesheets/_main.scss [hunk 1]
```diff
@@ -291,3 +291,8 @@ select {
 .cf-font-light {
   font-weight: 300;
 }
+
+// add to commons
+textarea {
+  margin-bottom: 40px;
+}
```
Comment (slj) [inline app/assets/stylesheets/_main.scss]: I feel that, in the future, things like margin and padding should be defined in `em` or `rem`.  But for now, this is cool b/c I'm sure the designers asked for 40px.

Comment (slj) [inline app/assets/stylesheets/_main.scss]: Besides, the designers requested it :)

File: client/app/containers/StyleGuide/StyleGuideTextArea.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import TextareaField from '../../components/TextareaField';
+
+export default class StyleGuideTextArea extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: ''
+    };
```
Comment (slj) [inline client/app/containers/StyleGuide/StyleGuideTextArea.jsx]: I think this is breaking a practice we've established where the state is managed/mutated outside the basic component.

**Comment by gnakm** [comment]
lgtm!

**Pull request merged**